ko/sql/schema/table_stmt.rst

```diff
@@ -793,10 +793,12 @@ DONT_REUSE_OID
 
     CREATE TABLE enc_tbl (a INT, b INT) ENCRYPT;
 
+암호화 여부는 상속되지 않는다.
```

Comment:
This information should also be added to the "클래스 상속" section. (cubrid-manual/ko/sql/oodb.rst)

Reply:
I'm not sure about this because there is no information about other table options in the "inheritance section" although they are not also. Or we can remove it for consistency with other options.

Reply:
@wrlawodms I don't think what you said is an important problem. I think anyone reading that section should be aware of the TDE constraints of inheritance.

Reply:
There are descriptions for method and attribute in the "클래스 상속", but no mention for others like the table option. The table option is not a property to be inherited, and, in my opinion, It means "TDE" information which is one of the table options is not inherited deductively. In addition, I can't find a context or phrase to explain it.
If it should be informed, we can put it in this Note below in the section. The comment can be "코멘트, reuse_oid, 암호화 여부 등의 테이블 옵션은 상속되지 않는다.", but I'm not sure the table option is the only one to be mentioned not to be inherited here
![image](https://user-images.githubusercontent.com/4846884/102449626-09d2df80-4078-11eb-83ca-4a5c19f566e2.png)

Reply:
Added: [9525d62](https://github.com/CUBRID/cubrid-manual/pull/250/commits/9525d62b2557e39be3e7814c7138671dac6ce3f0)

Reply:
@wrlawodms Please add only for TDE to Note.